src/tests/core/engine/processors/kichen_simulator_integration_test_v2.py

```diff
@@ -75,21 +75,24 @@ def test_assign_orders_to_available_chefs(self):
         )
 
         order_1 = build_order(order_id=1, status=OrderStatus.NEW_ORDER)
-        self.kitchen_simulator.mongo_order_status_history_controller.set_next_status_history_by_order_id(
+        kitchen_sim = self.kitchen_simulator
```

Comment:
Why don't you name it `kitchen_simulator`?

Reply:
done.